service/src/test/java/bio/terra/tanagra/service/FeatureSetServiceTest.java

```diff
@@ -299,20 +304,24 @@ void cloneFeatureSet() {
     assertNotNull(featureSet2);
     LOGGER.info("Created feature set {} at {}", featureSet2.getId(), featureSet2.getCreated());
 
-    // Clone the feature set without new description and verify
+    // Clone the feature set into different study without new description and verify
     String newDisplayName = "cloned feature set displayName";
     FeatureSet clonedFeatureSet2 =
         featureSetService.cloneFeatureSet(
-            study1.getId(), featureSet2.getId(), USER_EMAIL_2, newDisplayName, null);
-    assertEquals(
-        4,
-        featureSetService
-            .listFeatureSets(
-                ResourceCollection.allResourcesAllPermissions(
-                    ResourceType.FEATURE_SET, ResourceId.forStudy(study1.getId())),
-                0,
-                10)
-            .size());
+            study1.getId(),
+            featureSet2.getId(),
+            USER_EMAIL_2,
+            study2.getId(),
+            newDisplayName,
+            null);
+    List<FeatureSet> study2FeatureSets =
+        featureSetService.listFeatureSets(
+            ResourceCollection.allResourcesAllPermissions(
+                ResourceType.FEATURE_SET, ResourceId.forStudy(study2.getId())),
+            0,
+            10);
+    assertEquals(1, study2FeatureSets.size());
+    assertEquals(clonedFeatureSet2.getId(), study2FeatureSets.get(0).getId());
     assertEquals(newDisplayName, clonedFeatureSet2.getDisplayName());
     assertEquals("Copy of: " + featureSet2.getDescription(), clonedFeatureSet2.getDescription());
```

Comment:
When cloning into a different study, I think we should skip the "Copy of:" prefix when a new display name/description is not provided.

Reply:
updated